packages/components/src/composite/stories/index.story.tsx

```diff
@@ -11,35 +11,31 @@ import { isRTL } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import {
-	Composite,
-	CompositeGroup,
-	CompositeRow,
-	CompositeItem,
-	useCompositeStore,
-} from '..';
+import * as Composite from '..';
 import { UseCompositeStorePlaceholder, transform } from './utils';
 
 const meta: Meta< typeof UseCompositeStorePlaceholder > = {
-	title: 'Components/Composite (V2)',
+	title: 'Components/Composite',
 	component: UseCompositeStorePlaceholder,
 	subcomponents: {
 		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
-		Composite,
+		Root: Composite.Root,
 		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
-		CompositeGroup,
+		Group: Composite.Group,
```

Comment:
I think we'd want these to show the full dot-notated name, like we did for Tabs and CustomSelectControlV2.

```suggestion
		'Composite.Group': Composite.Group,
```